src/lib.rs

```diff
@@ -1,25 +1,55 @@
-extern "C" {
-    #![allow(improper_ctypes)] // we do not actually cross the FFI bound here
+use std::os::raw::c_char;
+use std::os::raw::c_int;
+use std::ffi::CString;
 
-    fn rust_fuzzer_test_input(input: &[u8]);
+extern "C" {
+    // This is the mangled name of the C++ function starting the fuzzer
+    fn _ZN6fuzzer12FuzzerDriverEPiPPPcPFiPKhmE(argc: *mut c_int, argv: *mut *mut *mut c_char, callback: extern fn(*const u8, usize) -> c_int );
```

Comment:
Is this the same on all platforms? Should we be exposing an extern c function instead?